docs/components/DynamicImports/Viewer.tsx

```diff
@@ -18,10 +18,12 @@ const CloverViewer = ({
   iiifContent = defaultIiifContent,
   options,
   customDisplays,
+  iiifContentSearch,
 }: {
   iiifContent: string;
   options?: ViewerConfigOptions;
   customDisplays?: Array<CustomDisplay>;
+  iiifContentSearch?: string;
```

Comment:
I think we can get by with just `q` for now but we might will want to define this type to fit the shape of https://iiif.io/api/search/2.0/#query-parameters as we extend functionality. This gives us room to grow.

```suggestion
  iiifContentSearchQuery?: {
    q: string;
  };
```